component/admin/models/payment.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * @package     Redcore.Backend
+ * @subpackage  Models
+ *
+ * @copyright   Copyright (C) 2008 - 2015 redCOMPONENT.com. All rights reserved.
+ * @license     GNU General Public License version 2 or later, see LICENSE.
+ */
+
+defined('_JEXEC') or die;
+
+/**
+ * Payment Model
+ *
+ * @package     Redcore.Backend
+ * @subpackage  Models
+ * @since       1.2
+ */
+class RedcoreModelPayment extends RModelAdmin
+{
+	/**
+	 * Prepare and sanitise the table data prior to saving.
+	 *
+	 * @param   JTable  $table  A reference to a JTable object.
+	 *
+	 * @return  void
+	 */
+	protected function prepareTable($table)
+	{
+		parent::prepareTable($table);
+
+		$now = JDate::getInstance();
+		$nowFormatted = $now->toSql();
+
+		if ((is_null($table->created_date) || empty($table->created_date)))
```

Comment:
it is not good enough in RTable :)
I do not have created_by so it wont fill my created_date